app/resources/admin.html.template

```diff
@@ -804,6 +804,27 @@
     </div>
   </fieldset>
 
+  <fieldset>
+    <legend>REPO</legend>
+    <div class="config">
+      <label for="repo_aliases">
+        REPO alias mapping.<br>
+        e.g., If it is
+        <code>{"jp": "japan", "ht": "haiti"}</code>, REPO
+        alias to jp are interpreted as requests for "japan"
```

Comment:
Can you clarify this explanation? For example,
e.g., If it is <code>{"jp": "japan", "ht": "haiti"}</code>, you can use http://google.org/personfinder/jp as an alias of http://google.org/personfinder/japan and the same for "ht". This is especially useful combined with the short URL e.g., http://g.co/pf/jp